JMMServer/Entities/TvDB_ImageFanart.cs

```diff
@@ -90,5 +90,16 @@ public bool Populate(int seriesID, XmlNode node)
 				return false;
 			}
 		}
+
+        public void Valid()

```

Comment:
but didn't you add it in this commit? or is it just whitespace and github diff sucks...